src/Behat/Behat/Output/Node/Printer/ExercisePrinter.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/*
+ * This file is part of the Behat.
+ * (c) Konstantin Kudryashov <ever.zet@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Behat\Behat\Output\Node\Printer;
+
+use Behat\Testwork\Output\Formatter;
+
+interface ExercisePrinter
```

Comment:
Define a new interface similar to the existing FeaturePrinter, ScenarioPrinter, etc...